lm_eval/models/huggingface.py

```diff
@@ -570,6 +573,25 @@ def _create_model(
             self._model = PeftModel.from_pretrained(
                 self._model, peft, revision=revision
             )
+        elif delta:
+            if autogptq:
+                eval_logger.warning(
+                    f"Delta weights might trigger unexpected behavior when used with AutoGPTQ."
+                )
+            _model_delta = self.AUTO_MODEL_CLASS.from_pretrained(
```

Comment:
Just want to make sure that `_model_delta`'s memory is freed up after its weights are added, and if it isn't, then adding `del _model_delta` would be handy